pkgs/tools/virtualization/google-compute-guest-agent/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, buildGoModule, makeWrapper, google-compute-guest-configs }:
+
+let
+  version = "20211019.00";
+
+in buildGoModule {
+  inherit version;
+  pname = "guest-agent";
```

Comment:
```suggestion
buildGoModule rec {
  inherit version;
  pname = "guest-agent";
  version = "20211019.00";
```

Reply:
version is used in the src section

Reply:
For that reason I added rec. This is the default recommended style which should be used in most cases and I don't see any reason to not use it in this package.